packages/cli/src/cmds/navie/navie.ts

```diff
@@ -134,9 +125,7 @@ function buildExplainRequest(
 export const handler = async (argv) => {
   verbose(argv.verbose);
   handleWorkingDirectory(argv.directory);
```

Comment:
Changing the working directory shouldn't be needed any more. We may want to update this navie one-shot CLI command to accept a directory list and propagate that via `configureRpcDirectories`.